src/index.js

```diff
@@ -29,7 +29,7 @@ class Timeout {
 
 const _setTimeout = (...args) => { return new Timeout(...args); };
 
-const _clearTimeout = (timer) => { return timer.close(); };
+const _clearTimeout = (timer) => { timer && timer.close && timer.close(); };
```

Comment:
I'm fine with keeping it this way. If you really want to be safe, you should change `timer.close` to `typeof timer.close === 'function'`. :-)

I guess you could go even further and verify that timer is a non-null object in the first place. Probably all overkill though.